ansible/deploy_http_proxy.yml

```diff
@@ -1,5 +1,5 @@
 - name: http proxy for external calls
-  hosts: all
+  hosts: patch_clients
```

Comment:
No other PR, actually we just need to exclude the patch server itself. I will look it up, something liek "all but patch"